postpic/helper.py

```diff
@@ -595,3 +595,81 @@ def kspace(component, fields, extent=None, interpolation=None, omega_func=None):
             result.matrix += (-1)**(i-1) * prefactor * mesh[mesh_i] * field.matrix
 
     return result
+
+
+def kspace_propagate(kspace, dt,
+                     moving_window=False, moving_window_vect=None,
+                     remove_antipropagating_waves=False):
```

Comment:
When only using `moving_window_vect=(1,0)`, users may wonder, why the window isnt moving. Maybe rename and set `move_window=None` (instead of `moving_window`) on default, which will move it if `moving_window_vect is not None` and not move it otherwise. If `move_window` is given additionally as `True` or `False` that will override the default. I think thats more intuitive to use.